alignak/basemodule.py

```diff
@@ -390,7 +408,14 @@ def _main(self):
         logger.info("Process for module %s is now running (pid=%d)", self.alias, os.getpid())
 
         # Will block here!
-        self.main()
+        try:
+            self.main()
+        except EOFError:
+            pass
+            # logger.warning('[%s] EOF exception: %s', self.alias, traceback.format_exc())
```

Comment:
Yes, it is a *usual* exception when you are reading a queue and the process is stopped ... no need to raise a log for this!